src/main/java/org/skife/jdbi/v2/sqlobject/BindBeanFactory.java

```diff
@@ -20,12 +20,12 @@
 import java.beans.PropertyDescriptor;
 import java.lang.annotation.Annotation;
 import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
 
 class BindBeanFactory implements BinderFactory
 {
     @Override
-    public Binder build(Annotation annotation)
-    {
+    public Binder build(Annotation annotation) {
```

Comment:
Careful with auto-formatting, it increases the chance of merge conflicts with no functional benefit.